### DISCUSSION TITLE
Ad Block Log - Unable to open log file

### DISCUSSION THREAD

**Discussion by magnetux** [RaspAP/raspap-webgui#1130] [General]
i have follow https://docs.raspap.com/adblock/#disabling-ad-block to enable ad block and log but 
RaspAP Ad Blocking > Logging > Unable to open log file

### Here is my Ad Blocking Logging Tab
![Screenshot 2022-03-01 142010](https://user-images.githubusercontent.com/51068408/156123573-79062867-f773-4cd5-9b15-6f740fffcaaf.png)

![Screenshot 2022-03-01 142043](https://user-images.githubusercontent.com/51068408/156123732-66958778-ef73-4b6f-91bd-e0cb5657e588.png)

### Here is my DHCP tab

![123](https://user-images.githubusercontent.com/51068408/156124244-15911597-021a-4e11-a040-d98cd60a12cb.png)

### /var/log/dnsmasq.log
```
# tail -n 20 /var/log/dnsmasq.log
Mar  1 07:16:14 dnsmasq[3235]: /etc/raspap/adblock/hostnames.txt w.usabilla.com is 0.0.0.0
Mar  1 07:16:14 dnsmasq[3235]: query[A] sync.opera.com from 192.168.100.246
Mar  1 07:16:14 dnsmasq[3235]: cached sync.opera.com is <CNAME>
Mar  1 07:16:14 dnsmasq[3235]: cached sync.geo.opera.com is <CNAME>
Mar  1 07:16:14 dnsmasq[3235]: cached nl.sync.opera.com is 185.26.182.111
Mar  1 07:16:14 dnsmasq[3235]: cached nl.sync.opera.com is 185.26.182.112
Mar  1 07:16:14 dnsmasq[3235]: query[A] tpc.googlesyndication.com from 192.168.100.246
Mar  1 07:16:15 dnsmasq[3235]: config tpc.googlesyndication.com is 0.0.0.0
Mar  1 07:16:15 dnsmasq[3235]: query[A] vrt.outbrain.com from 192.168.100.246
Mar  1 07:16:15 dnsmasq[3235]: config vrt.outbrain.com is 0.0.0.0
Mar  1 07:16:15 dnsmasq[3235]: query[A] w.usabilla.com from 192.168.100.246
Mar  1 07:16:15 dnsmasq[3235]: /etc/raspap/adblock/hostnames.txt w.usabilla.com is 0.0.0.0
Mar  1 07:16:15 dnsmasq[3235]: query[A] www.googletagservices.com from 192.168.100.246
Mar  1 07:16:15 dnsmasq[3235]: config www.googletagservices.com is 0.0.0.0
Mar  1 07:16:15 dnsmasq[3235]: query[A] www.ugdturner.com from 192.168.100.246
Mar  1 07:16:15 dnsmasq[3235]: config www.ugdturner.com is 0.0.0.0
Mar  1 07:16:15 dnsmasq[3235]: query[A] www.googletagservices.com from 192.168.100.246
Mar  1 07:16:15 dnsmasq[3235]: config www.googletagservices.com is 0.0.0.0
Mar  1 07:16:15 dnsmasq[3235]: query[A] www.ugdturner.com from 192.168.100.246
Mar  1 07:16:16 dnsmasq[3235]: config www.ugdturner.com is 0.0.0.0
```
seem ad blocking working but can't see log on RaspAp UI, can someone help me to fix this problem, or maybe i'm doing something wrong?

**Comment by billz** [comment]
This is a known issue that will be fixed in the next release.
If you'd like to patch this yourself see https://github.com/RaspAP/raspap-webgui/commit/0aad7c8f0e941a7210a7489b23b6c48736a5772c.
